nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/pom.xml

```diff
@@ -26,7 +26,7 @@
     <properties>
         <!--Wrapper Properties -->
         <nifi.jvm.heap.init>512m</nifi.jvm.heap.init>
```

Comment:
The minimum heap size should be changed to match the maximum heap size.

Reply:
Thanks for the review @exceptionfactory. Just out of curiosity, what is the reason for that? I thought a rule of thumb is to initially have the minimum as the 50% of the maximum heap size.

Reply:
Setting the minimum heap size to the same as the maximum makes the overall process memory usage more predictable. There are other factors that contribute to total process memory usage, but heap is usually the primary contributor. Starting out with the same amount of heap as the maximum avoids unexpected surprises when the heap grows to handle cached objects.